### DISCUSSION TITLE
how to update energy?

### DISCUSSION THREAD

**Discussion by JSchmidt63** [energye/energy#12] [Q&A]
hi,
i want to update the new a version of energy?

Like to, maybe?
```
import (
	"embed"
	//"fmt"
        "github.com/energye/energy/v2/cef"
	"github.com/energye/energy/consts"
)
```
and how to import in the go.mod? I am lacking the version in the go.mod.
Cheers

**Comment by sxmxta** [comment]
You can check the latest release version number

In mod, `require github. com/energy/energy/v2 [latest-version]`

latest-version: latest  release version number

run: go mod tidy
